pom.xml

```diff
@@ -326,16 +331,16 @@ Portions Copyright (c) 2018, 2020, Chris Fraire <cfraire@me.com>.
 
     <profiles>
         <profile>
-            <id>java9plus-8-release</id>
+            <id>java12plus-11-release</id>
```

Comment:
should we just get rid of these Java 9+ profiles ? there are similar entries in the other modules, e.g. indexer passes extra args when running tests, or suggester adds extra modules.

Reply:
I merged the `java9plus` profiles into the default, but this one here I believe is meant to set the `<release>` element to 11.

Reply:
Hmm, okay.